tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -49,29 +85,93 @@ describe('Header', () => {
     expect(menuBtn).toExist()
   })
 
-  it("should add usa-current class to Welcome tab when on '/edit-profile'", () => {
-    const store = mockStore(initialState)
+  it("should add usa-current class to Welcome tab when on '/welcome'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/welcome' } },
+    })
     const wrapper = mount(
       <Provider store={store}>
         <Header />
       </Provider>
     )
 
-    const welcomeTab = wrapper.find('.usa-nav__link')
+    const welcomeTab = wrapper.find('#welcome')
 
     expect(welcomeTab.hasClass('usa-current')).toEqual(true)
   })
 
+  it("should add usa-current class to Reports tab when on '/reports'", () => {
```

Comment:
Ensuring that the `Reports` navigation tab is active when on the `/reports` route